### DISCUSSION TITLE
Using `darkModeSelector` to always set dark mode

### DISCUSSION THREAD

**Discussion by ixxie** [primefaces/community#1672] [PrimeVue]
How do I force my theme to always use dark mode in the new styled mode of the 4.0 beta release?

I love the system-based toggle option, but I'm in the middle of a migration from another framework with a dark-only theme.

**Comment by Ssab1ro0v** [comment]
did you find a solution to this problem?

**Reply by ixxie** [reply]
Just hardcode the class to your body:

```html
<!-- index.html -->
<!doctype html>
<html lang="en" class="darkmode">
  <!-- etc -->
</html>
```

```js
// src/main.js
import PrimeVue from 'primevue/config';
import Aura from '@primevue/themes/aura';

const app = createApp(App);

// ...

app.use(PrimeVue, {
    // Default theme configuration
    theme: {
        preset: Aura,
        options: {
            darkModeSelector: '.darkmode',
        }
    }
 });
 
// ...
 ```

**Reply by glaysonsantos-fiepb** [reply]
Thanks, man. I wasted so much time looking for this solution. It worked perfectly here! But if I may ask, how did you came to this, any reference? I'm having a hard time learning how to use primevue 4 (I wonder what I am missing in the documentation...).

**Reply by ixxie** [reply]
> Thanks, man. I wasted so much time looking for this solution. It worked perfectly here! But if I may ask, how did you came to this, any reference? I'm having a hard time learning how to use primevue 4 (I wonder what I am missing in the documentation...).

I came to this by reasoning.... it took me a while but I realized in the end this is just about configure the class name which PrimeVue looks for when setting dark mode. Everything else you do yourself outside of the framework, including setting the class. The example in the docs is for the harder case of creating a dynamic dark mode toggle by adding or removing the class from the html document using Javascript, but you can also just hard code it.

PrimeVue learning is a bit tricky sometimes, I agree. Support is limited and the docs - while comprehensive - don't give all the answers. I recommend checking out the Prime Discord! Sometimes you will get answers there...
